modules/monitoring-about-configuring-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-configuring-core-platform-monitoring_{context}"]
+= About configuring core platform monitoring
+
+Edit the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace to configure the following core platform monitoring components: 
+
+[source,yaml]
+----
```

Comment:
I might be wrong, but this format of presenting comments and element names in a code block doesn't seem to follow established conventions. Unless you cite to other instances in openshift-docs, I would expect these files to be refused for merge.

For this type of reference material, I believe openshift-docs prefers the table format exemplified in `modules/bmo-about-the-baremetalhost-resource.adoc` and many other places. 

Another common reference format consists of an example definition file with callouts, as shown in `modules/cluster-autoscaler-cr.adoc`. This format is bulkier but shows users where to place the element (location, indentation) in the definition file, something that's missing from the tabular format.

I'd recommend using either or both of these established formats.